routing/router.go

```diff
@@ -155,9 +155,7 @@ func serveEnvJsFile(w http.ResponseWriter) {
 		body += fmt.Sprintf("window.HISTORY_MODE='%s';", conf.Server.WebHistoryMode)
 	}
 
-	if webRoot := strings.TrimSuffix(conf.Server.WebRoot, "/"); len(webRoot) > 0 {
-		body += fmt.Sprintf("window.WEB_ROOT='%s';", webRoot)
-	}
+	body += "window.WEB_ROOT = document.getElementsByTagName('base')[0].getAttribute('href').replace(/^https?:\\/\\/[^#?\\/]+/g, '').replace(/\\/+$/g, '')"
```

Comment:
Just for clarification and to document it here, can you explain what the regex is expected to do?

Here's my guess.

The first regex will replace any `http://<hostname>` or `https://<hostname>` with an empty string, leaving just the full path.

The second regex is the one originally proposed - it replaces everything in that full path, up to and including the last forward slash, with empty string.

Thus leaving the last path in the URL, which is assigned to WEB_ROOT.

@doflamingo721 @israel-hdez please confirm or comment with what we WANT those regexs to do. I assume we want WEB_ROOT to be the last path in the URL (??)

Reply:
> The first regex will replace any http://<hostname> or https://<hostname> with an empty string, leaving just the full path.

Yes that is correct.

Reply:
> The second regex is the one originally proposed - it replaces everything in that full path, up to and including the last forward slash, with empty string.
My understanding is that the second regex is replacing the forward slash at the end of the URL with an empty string thus removing it.